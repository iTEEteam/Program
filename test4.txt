loadmap testmap4.txt
make elf 2-1
make human 2-1
make dwarf 2-1
update 1
route Elf0 1
route Human0 0
route Dwarf0 3
update 1
drawMap
exit