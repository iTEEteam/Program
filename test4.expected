map testmap4.txt loaded
Elf0 created
Human0 created
Dwarf0 created
update
Elf0 moves
Elf0 crossroad
Human0 moves
Human0 crossroad
Dwarf0 moves
Dwarf0 crossroad
update
Elf0 moves
Human0 moves
Dwarf0 moves
	0	1	2	3	4	5
0	x	x	x	x	x	x
1	x	x	ho	r	r	x
2	en	r	r	hu	r	ex
3	x	x	el	r	r	x
4	x	x	x	x	x	x